CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# Rama Changelog
+All notable changes to this project will be documented in this file.
+
+## [v0.1.0] 2021-06-18
+### New features
+- First release version
+- Support overlay (vxlan) network
+- Support hybrid overlay/underlay container network
+- Full support for ipv4/ipv6 dual-stack
+- Node need only one physical nic if container network is in the same vlan with node network
```

Comment:
also i don't think this is a feature, maybe it's an instruction on a feature

Reply:
change to improvements